cmd/parca-agent/main.go

```diff
@@ -116,57 +128,32 @@ func main() {
 		}
 
 		wc = profilestorepb.NewProfileStoreServiceClient(conn)
-		dc = parcadebuginfo.NewDebugInfoClient(conn)
```

Comment:
I think this is unintentionally removed. With the current state we only use noopClient for the debuginfo.

Reply:
Resolved by https://github.com/parca-dev/parca-agent/pull/188